### PR TITLE
fix the nigjtly ItRemoteConsole failure

### PR DESCRIPTION
**Pull request by maggiehe00**
Full test suite passed at:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7331/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java [hunk 1]
```diff
@@ -161,10 +161,17 @@ void testWlsRemoteConsoleConnectionThroughTraefik() {
         "Could not get the default external service node port");
     logger.info("Found the Traefik service nodePort {0}", traefikNodePort);
     logger.info("The K8S_NODEPORT_HOST is {0}", K8S_NODEPORT_HOST);
-    String curlCmd = "curl -v --user weblogic:welcome1 -H Content-Type:application/json -d "
+    /*String curlCmd = "curl -v --user weblogic:welcome1 -H Content-Type:application/json -d "
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java:164]: No commented code

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java [hunk 2]
```diff
@@ -223,11 +237,19 @@ void testWlsRemoteConsoleConnectionUsingSSL() {
...
 
     //verify remote console is accessible through default-secure nodeport
-    curlCmd = "curl -sk -v --show-error --noproxy '*' --user weblogic:welcome1 -H "
+    /*curlCmd = "curl -sk -v --show-error --noproxy '*' --user weblogic:welcome1 -H "
         + "Content-Type:application/json -d "
         + "\"{ \\" + "\"domainUrl\\" + "\"" + ": " + "\\" + "\"" + "https://"
         + hostAndPort + "\\" + "\" }" + "\""
-        + " http://localhost:8012/api/connection  --write-out %{http_code} -o /dev/null";
+        + " http://localhost:8012/api/connection  --write-out %{http_code} -o /dev/null";*/
+    curlCmd = "curl -sk -v --show-error --noproxy '*' --user weblogic:welcome1 "
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java:245]: Can u make it as function and call with a parameter with treaefik/nginx port

**Review by anpanigr — COMMENTED** [review]
Remove the commented code and add a function to avoid duplicate code

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java (hunk 1 above)
Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java:164]: Yup, that is why I haven't requested for review :) Thanks Pani!

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java (hunk 2 above)
Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java:245]: Will do.

**Review by anpanigr — COMMENTED** [review]
Remove the hard-coded username/password  (weblogic/welcome1) from the file instead use variable from TestConstant class

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java [hunk 3]
```diff
@@ -223,11 +215,15 @@ void testWlsRemoteConsoleConnectionUsingSSL() {
...
-        + "Content-Type:application/json -d "
-        + "\"{ \\" + "\"domainUrl\\" + "\"" + ": " + "\\" + "\"" + "https://"
-        + hostAndPort + "\\" + "\" }" + "\""
-        + " http://localhost:8012/api/connection  --write-out %{http_code} -o /dev/null";
+    curlCmd = "curl -sk -v --show-error --noproxy '*' --user "
+        + ADMIN_USERNAME_DEFAULT + ":" + ADMIN_PASSWORD_DEFAULT
+        + " http://localhost:8012/api/providers/AdminServerConnection -H  "
+        + "\"" + "Content-Type:application/json" + "\""
+        + " --data "
+        + "\"{\\" + "\"name\\" + "\"" + ": " + "\\" + "\"" + "asconn\\" + "\"" + ", "
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java:223]: Can you put this in one string?

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java [hunk 4]
```diff
@@ -333,13 +329,35 @@ private static void verifyWlsRemoteConsoleConnection() {
...
-    String curlCmd = "curl -v --show-error --noproxy '*' --user weblogic:welcome1 -H Content-Type:application/json -d "
-        + "\"{ \\" + "\"domainUrl\\" + "\"" + ": " + "\\" + "\"" + "http://"
-        + hostAndPort + "\\" + "\" }" + "\""
-        + " http://localhost:8012/api/connection  --write-out %{http_code} -o /dev/null";
+    String curlCmd = "curl -v --show-error --noproxy '*' --user "
+        + ADMIN_USERNAME_DEFAULT + ":" + ADMIN_PASSWORD_DEFAULT
+        + " http://localhost:8012/api/providers/AdminServerConnection -H "
+        + "\"" + "Content-Type:application/json" + "\""
+        + " --data "
+        + "\"{\\" + "\"name\\" + "\"" + ": " + "\\" + "\"" + "asconn\\" + "\"" + ", "
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java:337]: same here

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java [hunk 5]
```diff
@@ -333,13 +329,35 @@ private static void verifyWlsRemoteConsoleConnection() {
...
 
+  private static void verifyRemoteConsoleConnectionThroughLB(int nodePortOfLB) {
+    logger.info("LB nodePort is {0}", nodePortOfLB);
+    logger.info("The K8S_NODEPORT_HOST is {0}", K8S_NODEPORT_HOST);
+
+    String curlCmd = "curl -v --user " + ADMIN_USERNAME_DEFAULT + ":" + ADMIN_PASSWORD_DEFAULT
+        + " http://localhost:8012/api/providers/AdminServerConnection -H "
+        + "\"" + "Content-Type:application/json" + "\""
+        + " --data "
+        + "\"{ \\" + "\"name\\" + "\"" + ": " + "\\" + "\"" + "asconn\\" + "\"" + ", "
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java:354]: same here

**Review by xiancao — CHANGES_REQUESTED** [review]

**Review by anpanigr — APPROVED** [review]
lgtm

**Review by xiancao — APPROVED** [review]
lgtm

**Review by mriccell — APPROVED** [review]

**Pull request merged**